src/test/java/de/tum/in/www1/artemis/quiz/QuizExerciseIntegrationTest.java

```diff
@@ -1526,36 +1066,27 @@ void importQuizExerciseFromCourseToCourseT() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void importQuizExerciseFromCourseToExam() throws Exception {
-        var now = ZonedDateTime.now();
-        Course course1 = database.addEmptyCourse();
-        ExerciseGroup exerciseGroup1 = database.addExerciseGroupWithExamAndCourse(true);
-        quizExercise = database.createQuiz(course1, now.plusHours(2), null, QuizMode.SYNCHRONIZED);
-        quizExerciseService.save(quizExercise);
-        quizExercise.setReleaseDate(null);
-        quizExercise.setCourse(null);
-        quizExercise.setDueDate(null);
-        quizExercise.setAssessmentDueDate(null);
-        quizExercise.setQuizBatches(new HashSet<>());
-        quizExercise.setExerciseGroup(exerciseGroup1);
+        ExerciseGroup exerciseGroup = database.createAndSaveActiveExerciseGroup(true);
+        QuizExercise quizExercise = database.createAndSaveQuiz(ZonedDateTime.now().plusHours(2), null, QuizMode.SYNCHRONIZED);
 
-        QuizExercise importedExercise = request.postWithResponseBody("/api/quiz-exercises/import/" + quizExercise.getId(), quizExercise, QuizExercise.class, HttpStatus.CREATED);
+        database.emptyOutQuizExercise(quizExercise);
+        quizExercise.setExerciseGroup(exerciseGroup);
 
-        assertThat(importedExercise.getExerciseGroup()).as("Quiz was imported for different exercise group").isEqualTo(exerciseGroup1);
+        QuizExercise importedExercise = request.postWithResponseBody("/api/quiz-exercises/import/" + quizExercise.getId(), quizExercise, QuizExercise.class, HttpStatus.CREATED);
+        assertThat(importedExercise.getExerciseGroup()).as("Quiz was imported for different exercise group").isEqualTo(exerciseGroup);
     }
 
     /**
      * test import quiz exercise to exam with invalid roles
      */
     @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "TA")
+    @WithMockUser(username = TEST_PREFIX + "editor1", roles = "TA")
```

Comment:
```suggestion
    @WithMockUser(username = TEST_PREFIX + "tutor1", roles = "TA")
```
considering the role

Reply:
good catch!